### PR TITLE
fix typo in sftp public key

### PR DESCRIPTION
**Pull request by MorrisJobke**
* downstream of https://github.com/owncloud/core/pull/27234

### CODE REVIEW

File: apps/files_external/lib/Controller/AjaxController.php [hunk 1]
```diff
@@ -73,7 +73,7 @@ public function __construct($appName,
 	private function generateSshKeys() {
 		$key = $this->rsaMechanism->createKey();
 		// Replace the placeholder label with a more meaningful one
-		$key['publicKey'] = str_replace('phpseclib-generated-key', gethostname(), $key['publickey']);
+		$key['publickey'] = str_replace('phpseclib-generated-key', gethostname(), $key['publickey']);
```
Comment (ChristophWurst) [inline apps/files_external/lib/Controller/AjaxController.php]: In terms of consistency, we use `publicKey` more often in our code base: https://github.com/owncloud/core/search?utf8=%E2%9C%93&q=%22%27publickey%27%22&type=Code

Comment (MorrisJobke) [inline apps/files_external/lib/Controller/AjaxController.php]: But this fixes an issue with wrong naming (see the name at the end of the line). Then we would need to rename a lot more AFAIK. But this needs to be evaluated by @icewind1991

Comment (icewind1991) [inline apps/files_external/lib/Controller/AjaxController.php]: `'publickey'` is [correct](https://github.com/nextcloud/server/blob/718d1f3f15ebe7429c7849ed41b0565ac0cda805/apps/files_external/lib/Controller/AjaxController.php#L91) here

**Review by icewind1991 — APPROVED** [review]

**Comment by MorrisJobke** [comment]
:+1: from me as well

**Pull request merged**